package.json

```diff
@@ -30,10 +30,12 @@
     "@rollup/plugin-commonjs": "24.1.0",
     "@rollup/plugin-node-resolve": "15.0.0",
     "@rollup/plugin-replace": "^5.0.0",
-    "@storybook/addon-knobs": "6.4.0",
-    "@storybook/addon-storyshots": "6.5.10",
-    "@storybook/addon-storysource": "6.5.10",
-    "@storybook/svelte": "6.5.10",
+    "@storybook/addon-controls": "^7.5.0",
+    "@storybook/addon-storyshots": "7.4.6",
+    "@storybook/addon-storysource": "7.4.6",
+    "@storybook/cli": "7.4.6",
+    "@storybook/svelte": "7.4.6",
+    "@storybook/svelte-webpack5": "^7.4.6",
```

Comment:
I strongly suggest we keep all storybook dependencies at the same version number, maybe 7.4.6, due to experiences in the past where CI broke because they were at different versions.

Reply:
```suggestion
    "@storybook/addon-controls": "7.4.6",
     "@storybook/addon-docs": "7.4.6",
    "@storybook/addon-storyshots": "7.4.6",
    "@storybook/addon-storysource": "7.4.6",
    "@storybook/cli": "7.4.6",
    "@storybook/svelte": "7.4.6",
    "@storybook/svelte-webpack5": "7.4.6",
```